Makefile

```diff
@@ -119,13 +119,13 @@ release-github:
 		-v /var/run/docker.sock:/var/run/docker.sock \
 		-w /go/tmp \
 		--env GORELEASER_GITHUB_TOKEN \
-		neilotoole/xcgo:latest goreleaser release --rm-dist
+		ghcr.io/gythialy/golang-cross:latest --cleam
```

Comment:
ah yes, typo indeed
`goreleaser` isn't needed since it's the default command for the base image: https://github.com/gythialy/golang-cross/blob/main/entrypoint.sh (and` --rm-dist` is deprecated by newer goreleaser version in favor of `clean`)